app/scripts/proactive/model/utils.js

```diff
@@ -18,13 +18,17 @@ define(function (require) {
         },
         inlineNameValueInherited: function(prop) {
         	var checked = "";
+        	// update datatype if wrong, otherwise dislay is not correct
+        	if (prop['Inherited'] == "true") prop['Inherited'] = true;
+        	if (prop['Inherited'] == "false") prop['Inherited'] = false;
+
         	if (prop['Inherited']){
         		checked=" checked"
         	}
```

Comment:
Here, can we use if(prop['Inherited']=="true"), instead of change it, if we don't need to use it later as a boolean? Then we can save 2 lines.